documentation/site/pages/provers/quick-start.mdx

```diff
@@ -31,101 +35,157 @@ For a quick set up of Boundless dependencies on Ubuntu 22.04 LTS (see [Operating
 sudo ./scripts/setup.sh
 ```
 
-## Running Boundless
-
-The recommended place to start are the `just` commands. This script will run everything you need to start proving.
+## Running a Test Proof
 
 Boundless is comprised of two major components:
-1. **Bento** is the local proving infrastructure. Bento will take requests, prove them and return the result
-2. The **Broker** interacts with the Boundless market. Broker can submit or request proves from the market.
+1. *Bento* is the local proving infrastructure. Bento will take requests, prove them and return the result
+2. The *Broker* interacts with the Boundless market. Broker can submit or request proves from the market.
 
-To get started, run:
+To get started with a test proof on a new proving machine, you'll need to install the `bento_cli`:
 
 ```sh [Terminal]
-just broker
+cargo install --git https://github.com/risc0/risc0 bento-client --bin bento_cli
 ```
+Once installed, you can run bento with:
 
-To stop all services, run:
+```sh [Terminal]
+just bento
+```
+
+This will spin up bento without the broker. You can check the logs at any time with:
 
 ```sh [Terminal]
-just broker down
+just bento logs
 ```
 
-Or remove all volumes and data from the service:
+To run the test proof:
 
 ```sh [Terminal]
-just broker clean
+RUST_LOG=info bento_cli -c 32
 ```
 
----
+If everything works, you should see something like the following:
 
-And to run with a custom environment file:
+![Bento CLI Test Proof Success](/test-proof.png)
+
+Before we move on, stop Bento with:
 
 ```sh [Terminal]
-just broker up ./.env.broker
-just broker down ./.env.broker
+just bento down
 ```
```

Comment:
Technically you don't need to stop bento before running the broker, and when you just just broker it will just start the broker and connect to this existing node.